packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -212,6 +213,14 @@ export class FauxClassGenerator {
     return this.successExit();
   }
 
+  // VisibleForTesting
+  public isRequiredSObject(sobject: string): boolean {
+    if (sobject.endsWith('Share') || sobject.endsWith('History') || sobject.endsWith('Feed') || sobject.endsWith('Event')) {
+      return false;
+    }
+    return true;
```

Comment:
```suggestion
    return !/Share$|History$|Feed$|Event$/.test(sobject);
```

Reply:
I generally avoid regex - it makes readability harder and difficult to maintain. Any thoughts others?

Reply:
@xyc - I've updated to the regex with some description about what we are doing. Let me know if it looks ok.